File: app/src/main/java/com/example/foundlah/Convo.java
package com.example.foundlah;

import java.util.List;

public class Convo {
    private String user1;
    private String user2;
    private String chatroomID;

    public Convo(){}

    public String getUser1(){
        return user1;
    }

    public String getUser2(){
        return user2;
    }

    public void setUsers(List<String> users){
        if (users != null && users.size() == 2) {
            this.user1 = users.get(0);
            this.user2 = users.get(1);
        } else {
            this.user1 = "Unknown";
            this.user2 = "Unknown";
        }
    }

    public String getChatroomID(){
        if(user1.compareTo(user2) < 0){
            return user1 + "_" + user2;
        }
        else{
            return user2 + "_" + user1;
        }
    }
}

File: app/src/main/java/com/example/foundlah/Convo.kt
package com.example.foundlah

class Convo {
    var user1: String? = null
        private set
    var user2: String? = null
        private set
    val chatroomID: String?
        get() = if (user1!!.compareTo(user2!!) < 0) {
            user1 + "_" + user2
        } else {
            user2 + "_" + user1
        }

    fun setUsers(users: List<String?>?) {
        if (users != null && users.size == 2) {
            this.user1 = users[0]
            this.user2 = users[1]
        } else {
            this.user1 = "Unknown"
            this.user2 = "Unknown"
        }
    }
}
